kolena/initialize.py

```diff
@@ -96,3 +136,20 @@ def initialize(
         # Configure third party logging based on verbosity
         pd.set_option("display.max_colwidth", None)
         log.info(f"connected to {get_platform_url()}")
+
+
+def _find_token() -> str:
+    if KOLENA_TOKEN_ENV in os.environ:
+        return os.environ[KOLENA_TOKEN_ENV]
+
+    hostname = urlparse(state._get_api_base_url()).hostname
+    try:
+        netrc_file = netrc.netrc()
+        record = netrc_file.authenticators(hostname)
```

Comment:
Do we have any process of adding the token to the `.netrc` file? I would hope to do so automatically on successful initialize.

Reply:
Not currently and I don't believe that we should (at least by default). Some users may not want their api token written to disk for security reasons.